UnityProject/Assets/Scripts/Systems/Faith/FaithProperties/RockAndStone.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Systems.Score;
+using UnityEngine;
+
+namespace Systems.Faith.FaithProperties
+{
+	public class RockAndStone : IFaithProperty
+	{
+		private string faithPropertyName = "Rock and Stone";
+		private string faithPropertyDesc = "Channel your inner dwarf. Mine the earth.";
+		[SerializeField] private Sprite propertyIcon;
+
+		string IFaithProperty.FaithPropertyName
+		{
+			get => faithPropertyName;
+			set => faithPropertyName = value;
+		}
+
+		string IFaithProperty.FaithPropertyDesc
+		{
+			get => faithPropertyDesc;
+			set => faithPropertyDesc = value;
+		}
+
+		Sprite IFaithProperty.PropertyIcon
+		{
+			get => propertyIcon;
+			set => propertyIcon = value;
+		}
+
+		public void Setup()
+		{
+			ScoreMachine.Instance.OnScoreChanged.AddListener(UpdatePoints);
+		}
+
+		private void UpdatePoints(string ID, int points)
+		{
+			if (ID != RoundEndScoreBuilder.COMMON_SCORE_LABORPOINTS) return;
+			FaithManager.AwardPoints(points);
+		}
+
+		public void OnJoinFaith(PlayerScript newMember)
+		{
+			//Work on adding beard additions when joining this faith.
+			//(Max): Adding body parts and changing sprites for them on the player is still too difficult and unreliable, Bod.
```

Comment:
the adding should be okay, I can have a look at the changing sprite stuff?

Reply:
an api that allows us to quickly add, remove and change sprites from body parts would be super helpful.

Reply:
I shouldn't have to do a bunch of checks every time I want to change anything with player sprites, I should just be able to say "hey, add this sprite SO" or "change this specific sprite's colors" easily 
also the body part enums for sprites are still broken so doing this reliably is out of the question